Doc/library/argparse.rst

```diff
@@ -192,6 +192,12 @@ arguments it contains. The default message can be overridden with the
 The ``%(prog)s`` format specifier is available to fill in the program name in
 your usage messages.
 
+When a custom usage message is specified for the main parser, you may also want to
+consider passing  the ``prog`` argument to :meth:`~ArgumentParser.add_subparsers`
+or the ``prog`` and the ``usage`` arguments to
+:meth:`~_SubParsersAction.add_parser`, to ensure consistent command prefixes and 
```

Comment:
This was merged with a failing lint check on the CI. 

https://github.com/python/cpython/actions/runs/11701457525/job/32587601941

Please could you trim the trailing space here?
```diff
-:meth:`~_SubParsersAction.add_parser`, to ensure consistent command prefixes and 
+:meth:`~_SubParsersAction.add_parser`, to ensure consistent command prefixes and
```

Reply:
Or maybe not.

Reply:
#127162.